packages/map-template/src/components/Search/Categories/Categories.jsx

```diff
@@ -57,6 +61,10 @@ function Categories({ onSetSize, getFilteredLocations, searchFieldRef, isOpen })
 
     const [categoriesWithoutChildKeys, setCategoriesWithoutChildKeys] = useState([]);
 
+    const [categoriesWithChildKeys, setCategoriesWithChildKeys] = useState([]);
```

Comment:
Is it necessary to have separate states for the categories to show (categoriesWithoutChildKeys, categoriesWithChildKeys) instead of treating them as one state, eg. `categoriesToShow` where the content is decided by the topLevelCategory prop?

This would have the benefits of:
- an opportunity to simplify the JSX by removing some duplicate code
- an opportunity to have only one useEffect on the categories state